lib/itamae/recipe.rb

```diff
@@ -80,6 +80,12 @@ def method_missing(*args, &block)
       end
 
       def define(name, params = {}, &block)
+        class_name = "Itamae::Resource::#{Resource.get_resource_class_name(name)}"
+        if Module.const_defined?(class_name) && Module.const_get(class_name).is_a?(Class)
```

Comment:
Thank you for review!
I tried to reflect. Please check it.
(and add integration-test)